x/market/hooks/hooks.go

```
@@ -41,17 +41,29 @@ func (h *hooks) OnEscrowAccountClosed(ctx sdk.Context, obj etypes.Account) {
+		h.dkeeper.CloseDeployment(ctx, deployment)
 	}
 
 	for _, group := range h.dkeeper.GetGroups(ctx, deployment.ID) {
-		if group.ValidateClosable() == nil {
-			_ = h.dkeeper.OnCloseGroup(ctx, group, gstate)
-			_ = h.mkeeper.OnGroupClosed(ctx, group.ID)
+		switch gstate {
+		case dtypes.GroupPaused:
+			if group.ValidatePausable() == nil {
+				_ = h.dkeeper.OnPauseGroup(ctx, group)
+				h.mkeeper.OnGroupClosed(ctx, group.ID)
```

golangci-lint (2.5.0)

[error] Error return value of `h.mkeeper.OnGroupClosed` is not checked

(errcheck)